getting_started/developers_cli.adoc

```diff
@@ -200,83 +216,92 @@ project namespace.
 
 [[developers-cli-creating-an-application]]
 == Creating an Application
-To create a new application from the Instant App template
-ifdef::openshift-enterprise,openshift-origin,openshift-dedicated[]
-that you uploaded
-endif::[]
-:
+To create a new application from the code in your forked repository:
 
+. Create the application by specifying the source of the code:
++
 ----
-$ oc new-app nodejs-mongodb-example
+$ oc new-app openshift/ruby-20-centos7~https://github.com/<your_GitHub_username>/ruby-ex
```

Comment:
Maybe all lowercase for `github` here. Only mentioning because I believe we usually keep replaceables all lowercase.